pf9/express.py

```diff
@@ -42,7 +42,7 @@ def cli(ctx):
 # Any commands defined here or added will be toplevel
 
 
-cli.add_command(version)
```

Comment:
Are we just commenting this? And not actually deleting the code ?

Reply:
Yes. That code is a bit convoluted to understand which setup steps need to be cleaned up. I just commented this to not change too much for this trivial issue.